ratings/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Widget.Vitamin.RatingView">
+        <item name="vtmnRatingView_noteTextAppearance">@style/TextAppearance.Vitamin.Ratings.Note</item>
+        <item name="vtmnRatingView_commentsCountTextAppearance">@style/TextAppearance.Vitamin.Ratings.Comments</item>
+        <item name="vtmnRatingView_defaultIconsColor">?attr/vtmnContentPrimary</item>
+        <item name="vtmnRatingView_emphasisIconsColor">?attr/vtmnContentActive</item>
+        <item name="vtmnRatingView_compactIcon">@drawable/ic_vtmn_star_fill</item>
+    </style>
+
+    <style name="Widget.Vitamin.RatingView.RatingBar" parent="Widget.AppCompat.RatingBar.Small">
```

Comment:
What do you think about `Widget.Vitamin.RatingView.Bar`? It works for AppCompat because there isn't `Rating` before the `RatingBar` keyword.

Reply:
Changed